src/test/scala/io/iohk/ethereum/blockchain/sync/BlockBroadcastSpec.scala

```diff
@@ -33,7 +34,26 @@ class BlockBroadcastSpec
       NewBlock(Block(blockHeader, BlockBody(Nil, Nil)), initialPeerInfo.chainWeight.increaseTotalDifficulty(2))
 
     //when
-    blockBroadcast.broadcastBlock(newBlock, Map(peer -> initialPeerInfo))
+    blockBroadcast.broadcastBlock(BlockToBroadcast(newBlock.block, newBlock.chainWeight), Map(peer -> initialPeerInfo))
```

Comment:
maybe we could add in the end of the test description: `(PV64)` ->

`send a new block when it is not known by the peer (known by comparing chain weights) (PV64)`